src/filed.py

```diff
@@ -82,7 +80,7 @@ def load_config_file(filename):
 
 def main():
     config_files = [
-        '~/.filedrc'
+        path.join(path.dirname(__file__), 'resource/filedrc')
```

Comment:
Oh. I should have mentioned that was just to wire up the example configure file.

We still need to figure out how we want to lay out the confit file.